### PR TITLE
Added support quantize activations with ignored scopes in propagation mode

### PR DESCRIPTION
**Pull request by asenina**

### CODE REVIEW

**Comment by asenina** [comment]
@vshampor

**Comment by vshampor** [comment]
Jenkins please retry a build

**Comment by vshampor** [comment]
@asenina please post the new illustrations for the barrier-enabled graph here as well.

**Comment by asenina** [comment]
final
start

**Comment by vshampor** [comment]
@asenina please also post a config section that was used to setup these exact barrier locations?

**Comment by asenina** [comment]
@vshampor  `ignored_scopes = ['SmallTestNet/Sequential[block1]']`

File: nncf/nncf_network.py [hunk 1]
```diff
@@ -138,6 +138,7 @@ def close(self):
 class InsertionPointGraphNodeType(Enum):
     INSERTION_POINT = 0
     OPERATOR = 1
+    AUXILIARY_BARRIER = 2
```
Comment (vshampor) [inline nncf/nncf_network.py:141]: Should be moved to a proper enum type

**Comment by asenina** [comment]
Jenkins please retry a build

File: nncf/quantization/quantizer_propagation.py [hunk 2]
```diff
@@ -144,12 +153,43 @@ def __init__(self, ip_graph: InsertionPointGraph):
...
 
+        for u_node_key, v_node_key in barrier_node_extra_edges:
+            edge_attr = {QuantizerPropagationStateGraph.AFFECTING_PROPAGATING_QUANTIZERS_ATTR: []}
+            next_v_node_key = list(self.succ[v_node_key].keys())[0] # POST HOOK v
+            self.add_edge(v_node_key, u_node_key, **edge_attr)
+            self.add_edge(u_node_key, next_v_node_key, **edge_attr)
+            self.remove_edge(v_node_key, next_v_node_key)
+
+    @staticmethod
+    def ipg_node_type_to_qpsg_node_type(ipg_node_type: InsertionPointGraphNodeType):
```
Comment (vshampor) [inline nncf/quantization/quantizer_propagation.py:182]: Add a return value type hint.

**Comment by vshampor** [comment]
Jenkins please retry a build

**Comment by vshampor** [comment]
@asenina let's see the updated .png files with the same ignored scope config as before.

**Comment by asenina** [comment]
@vshampor 
final_v1

start_v1

**Comment by vshampor** [comment]
Jenkins please retry a build

**Comment by asenina** [comment]
Jenkins please retry a build

**Comment by vshampor** [comment]
Jenkins please retry a build

**Comment by vshampor** [comment]
Jenkins please retry a build

**Pull request merged**